srcpkgs/urlwatch/template

```diff
@@ -11,11 +11,11 @@ depends="python3-appdirs python3-keyring python3-minidb python3-requests
 checkdepends="python3-pytest python3-pycodestyle python3-docutils
  python3-Pygments ${depends}"
```

Comment:
Add `jq` here (or even in `depeds`) and that part should work.

https://urlwatch.readthedocs.io/en/latest/dependencies.html lists some other optional dependencies.

Reply:
That doesn't work though. What it needs is the jq Python bindings [jq.py](https://github.com/mwilliamson/jq.py), not `jq` itself. We can add it as a new package.